extractors/JSONPathExtractor.go

```diff
@@ -22,40 +22,17 @@ func (instance JSONPathExtractor) Extract(result core.ExecutionResult) core.Extr
 		"scope": instance.Scope,
 	}
 
-	paths, err := jsonpath.ParsePaths(instance.JSONPath)
-	if err != nil {
-		fmt.Println(fmt.Sprintf("Error Parsing %v", err))
-		return extractionResult
-	}
-
-	fmt.Println(fmt.Sprintf("Parsing path %s", instance.JSONPath))
-	eval, err := jsonpath.EvalPathsInBytes([]byte{}, paths)
-	fmt.Println(fmt.Sprintf("Eval %v", eval))
+	if data, ok := result[instance.Key]; ok {
+		var json_data interface{}
+		json.Unmarshal([]byte(data.(string)), &json_data)
+		res, err := jsonpath.JsonPathLookup(json_data, instance.JSONPath)
+		if err != nil {
+			extractionResult[instance.Name] = "Unexpected error evaluating JSON Path"
+		}
 
-	if err != nil {
-		fmt.Println(fmt.Sprintf("Error Evaluating %v", err))
-		return extractionResult
+		extractionResult[instance.Name] = res
```

Comment:
Don't we need an else on the `if err != nil`? If I am reading the right we will set the `extractionResult[instance.Name]` to be the `"Unexpected error evaluating JSON Path"` and then overwrite it with `res`.